keras/src/models/variable_mapping.py

```diff
@@ -2,25 +2,25 @@
 from keras.src.metrics.metric import Metric
 from keras.src.optimizers.optimizer import Optimizer
 from keras.src.saving import saving_lib
-from keras.src.saving.keras_saveable import KerasSaveable
+from keras.src.saving.keras_savable import KerasSavable
```

Comment:
"Saveable" is also a correct spelling and we use it a lot, so I recommend we keep it across the board.

Reply:
Pedantry! - Ok fine made the edit :gb: ^_^